streaming/src/test/scala/org/apache/spark/streaming/MapWithStateSuite.scala

```diff
@@ -64,6 +61,9 @@ class MapWithStateSuite extends SparkFunSuite
     } finally {
       super.afterAll()
     }
+    if (checkpointDir != null) {
```

Comment:
This should be inside the try block, after stopping the context. Is it safe to share this dir across all tests though?